File: QueuesAndStacks/StaTestStack.java
import java.util.*;

public class StaTestStack {
    int s[] = new int[10];

    public static void main(String[] args) {
        Stack<Integer> sts = new Stack<>();

        sts.push(15);
        sts.push(20);
        sts.push(35);
        sts.forEach(System.out::println);
        System.out.println(sts.peek());
        sts.pop();
        System.out.println(sts.peek());
        System.out.println(sts.size());
        // sts.pop();
        sts.push(5);
        sts.forEach(System.out::println);
        System.out.println(sts.peek());
        System.out.println(sts.isEmpty());

        sts.forEach(System.out::println);
        System.out.println(sts.isEmpty());
        Integer pos = sts.search(15);
        System.out.println(pos);
    }
}

File: QueuesAndStacks/staTestStack.js
const staTestStack = () => {

    let s = [];

    s.push(15);
    s.push(20);
    s.push(35);
    console.log(s[s.length - 1]);
    s.pop();
    console.log(s[s.length - 1]);
    console.log(s.length);
    s.push(5);
    console.log(s[s.length - 1]);
    s.pop();
    s.pop();
    console.log(s);
    s.pop();
    console.log(s.length < 1 ? true : false);
}

staTestStack();